### DISCUSSION TITLE
Plausible configuration button navigates to /[object%20Object]

### DISCUSSION THREAD

**Discussion by liliakai** [GitbookIO/community#794] [0. General bugs and questions]
See screenshot. Clicking the WHITE configure button navigates to 
https://app.gitbook.com/o/<orgID>/integrations/plausible/configuration/site/site_ZWiTf

Clicking the GREEN configure button navigates to
https://app.gitbook.com/o/<orgID>/integrations/plausible/configuration/site/[object%20Object]

<img width="1003" alt="Screenshot 2024-11-08 at 19 15 22" src="https://github.com/user-attachments/assets/a2ed75ca-1bfa-4d75-84d5-9c93d1051fe1">

👏 ... 👏  ... 👏

**Comment by emmerich** [comment]
Hi @liliakai thanks for the report - a fix is on the way and will be deployed within the hour.

**Comment by liliakai** [comment]
This one is fixed. ✅
